lib/elixir/lib/string.ex

```diff
@@ -1915,6 +1914,9 @@ defmodule String do
       iex> String.to_float("3.0")
       3.0
 
+      String.to_float("3")
```

Comment:
Should this be a doctest like the other examples (see [L1859](https://github.com/tsubery/elixir/blob/cc6d9abf3f5f3de38f99b3e80877cece6ecef5af/lib/elixir/lib/string.ex#L1859))?